README.md

```diff
@@ -123,3 +215,78 @@ cd ../..
   * `npm run generate-3d-tiles`
   * `npm run generate-quantized-mesh-terrain`
 * On Windows, the line endings of the generated files will be different than those checked into the repo. Just `git add` them and git will fix the line endings (no need to commit).
+
+#### Regenerate Dependency Graphs
+
+The dependency graphs used in the Cesium Native documentation are generated using a script that parses CMake's GraphViz output and generates a set of Mermaid diagrams. To regenerate the graphs:
+
+* Make sure you have a relatively recent version of Node.js installed.
+* Install dependencies by running:
+```
+cd tools/dep-graph-gen
+npm install
+```
+* From the `tools/dep-graph-gen` directory, run `npm run generate-dep-graph` to regenerate the graphs.
+
+## 🤝Contributing
```

Comment:
Could the "Contributing" sections be moved to a `CONTRIBUTING.md` file in the `doc` folder, similar to how it's organized in CesiumJS? We can still link to it similar to how CesiumJS's readme does [here](https://github.com/CesiumGS/cesium?tab=readme-ov-file#what-next)